WordPress/Classes/Services/UserService.swift

```diff
@@ -5,13 +5,11 @@ import WordPressUI
 
 /// UserService is responsible for fetching user acounts via the .org REST API – it's the replacement for `UsersService` (the XMLRPC-based approach)
 ///
-actor UserService: UserServiceProtocol {
+actor UserService: UserServiceProtocol, UserDataStoreProvider {
     private let client: WordPressClient
 
     private let _dataStore: InMemoryUserDataStore = .init()
-    var dataStore: any UserDataStore {
-        _dataStore
-    }
+    var userDataStore: any UserDataStore { _dataStore }
```

Comment:
Do we need the extra indirection here? Could `userDataStore` just be the canonical object?

Reply:
When declaring it as `var userDataStore: InMemoryUserDataStore`, the compiler complains that the signature does not match the one in `UserServiceProtocol`.